marimbabot_vision/scripts/flatten_whiteboard_data.py

```diff
@@ -0,0 +1,25 @@
+import os
+import tqdm
+from multiprocessing import Pool
+import re
+from shutil import copyfile
+
+NUM_WORKER = 30
+INPUT_DIR = 'data'
+OUTPUT_DIR = 'data_wb'
+NAME_PREFIX = 'staff_1'
+
+def main(sample):
```

Comment:
I would prefer to rename this to a more descriptive name

Reply:
You are right, I was a bit lazy here. I changed the name of this function to 'generate_sample_folder'.